src/index.js

```diff
@@ -54,6 +55,7 @@ export default class JSONTree extends React.Component {
       PropTypes.object
     ]).isRequired,
     hideRoot: PropTypes.bool,
+    isNotAllowExtractNestedItemOfList: PropTypes.bool,
```

Comment:
This is a generic library. It shouldn't know about our use case.

Instead, change this to maxClickableNodeDepth.

We will set it to 1 in our code if the data is a list.